package.json

```diff
@@ -71,26 +71,26 @@
     "phantomjs-prebuilt": "~2.1.4",
     "serve-favicon": "~2.3.0",
     "socket.io": "^1.4.8",
-    "validator": "~6.0.0",
+    "validator": "~6.1.0",
     "winston": "^2.2.0",
     "wiredep": "~4.0.0"
   },
   "devDependencies": {
     "coveralls": "~2.11.6",
     "del": "^2.2.2",
-    "eslint": "~2.2.0",
-    "eslint-config-airbnb": "~6.0.2",
+    "eslint": "~3.9.1",
+    "eslint-config-airbnb": "~13.0.0",
```

Comment:
This is a big bump and causing a lot of linting errors. Any insight into why? Have the linting rules changed that much?

Perhaps new rules have been added with defaults that we are violating?